exporter/containerimage/common.go

```diff
@@ -23,6 +26,10 @@ const (
 	preferNondistLayersKey = "prefer-nondist-layers"
 )
 
+var (
+	keyAnnotationRegexp = regexp.MustCompile(`^(?:(\S+)\.)?annotation(?:-(\S+))?\.(\S+)$`)
```

Comment:
Let's keep this as simple as possible. All annotations need to start with `annotation-` key. For ref based maybe `annotation-manifest[linux/amd64].foo`

Reply:
Agreed, I like that syntax a lot more :tada: